### DISCUSSION TITLE
A stopped tests Duration still increases in Metadata

### DISCUSSION THREAD

**Discussion by jameswilliams-iml** [artilleryio/artillery#3416] [Artillery]
Morning team,
After stopping a test, I can see in the Metadata top right of the test, the "Duration" of the test is still counting up. There's a few tests I stopped and this is happening on. Link to one below which currently is showing a duration of 3d 19h 39m 39s — stopped by jameswilliams-iml

https://app.artillery.io/share/sh_b8618109bdbaca6b0e090d49cc2f40e1e59c78e3b2a96d61e79516a8f2e8ec60

Shouldn't this also stop counting when the test is stopped/cancelled?
Thanks

**Comment by hassy** [comment]
thanks @jameswilliams-iml! taking a look

**Reply by hassy** [reply]
just deployed the fix, thanks for flagging it up @jameswilliams-iml!

**Reply by jameswilliams-iml** [reply]
Amazing @hassy . No problem, and thanks for the speedy solution!
